### PR TITLE
Performance

### PR DESCRIPTION
**Pull request by imaffett**
Broke out the files into the structure.  Still need to do base CSS classes, but that can come later.  This should be able to get you going now.  You can run the php scripts to "compile" the files in ui/jq.ui.js and ui/transitions/all.js

### CODE REVIEW

File: kitchensink/jq.ui.css [hunk 1]
```diff
@@ -70,8 +68,7 @@ width:100%;
 	width:100%;	height:1px;
 } /* Horizontal line. */
 
-.jqmScrollPanel { width:100%; min-height:100%; }
```
Comment (carlosouro) [inline kitchensink/jq.ui.css:73]: I've added this in to fix a bug in Android - jqmScrollPanel is the lower panel on the 2 panel JS scrolling.
Is is necessary to remove?

File: kitchensink/jq.ui.css [hunk 2]
```diff
@@ -807,8 +819,9 @@ select[disabled]~div {
 
 	
 .hasMenu{
+-webkit-transition: all 300ms;
```
Comment (carlosouro) [inline kitchensink/jq.ui.css:822]: Why is this here?
If we want to change transitions to 300 ms, we should do it in the UI (maybe a param?) where the animations are handled, this here might cause some iPhone "blurryness" - this is the initial reason why we removed it.

File: kitchensink/jq.ui.css (hunk 1 above)
Comment (imaffett) [inline kitchensink/jq.ui.css:73]: that was my fault - bad copy/paste job

File: kitchensink/jq.ui.css (hunk 2 above)
Comment (imaffett) [inline kitchensink/jq.ui.css:822]: That can be removed.  -webkit-transform:translate3d is generally what causes the blurryness on iOS though.

Comment (carlosouro) [inline kitchensink/jq.ui.css:822]: Ok, Already merged the pull request and fixed the commented changes. You can pull to keep on working :)

Sent on the walk
